README.md

```diff
@@ -1,3 +1,22 @@
+# 쿠버네티스 문서 - 한글 번역
+
+이 저장소는 쿠버네티스 문서의 한글 번역을 위한 저장소입니다.
+
+## 커뮤니티, 논의, 컨트리뷰션 및 지원
+
+쿠버네티스 커뮤니티와 함께 일하는 방법을 [커뮤니티 페이지](http://kubernetes.io/community/)에서 배우세요.
+
+다음의 연락처를 통해 이 프로젝트의 메인테이너에게 연락할 수 있습니다.
+
+- [Slack](http://slack.k8s.io/)
```

Comment:
How about "#kubernetes-docs-ko channel in [Slack - Kubernetes ](http://slack.k8s.io)"?